src/main/java/frc/robot/commands/BangBangDriveCommand.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.DriveConstants;
+import frc.robot.subsystems.Drivetrain;
+import java.awt.geom.Point2D;
+
+public class BangBangDriveCommand extends Command{
+    Drivetrain driv;
+    Pose2d setpoint;
+    int tim;
```

Comment:
Make sure to use private when creating variables like driv, it automatically saves as package-private if you don't specify, but you want restricted access to the class where it's declared

Reply:
Yeah. I would also recommend not using abbreviations unless they're super clear or are standard conventions.

Reply:
You changed some to protected, but no command should extend this (TurnyCommand does something completely different, and the instructions that said to extend your command refer to the one for your subsystem that you were supposed to make).

Reply:
Make this private instead.